src/ripple/app/paths/Pathfinder.cpp

```diff
@@ -92,6 +92,23 @@ bool comparePathRank (
     return a.index > b.index;
 }
 
+// Compare two PathRanks based on liquidity.
+// A better PathRank is lower, so the best are sorted to the beginning.
+bool comparePathLiquidity(
+    Pathfinder::PathRank const& a, Pathfinder::PathRank const& b)
```

Comment:
Because of the overhead of calling another function, yes? That's a good reason, and I didn't think of it at the time.